resources/assets/js/components/PostItem.vue

```diff
@@ -1,15 +1,17 @@
 <template>
     <tr>
         <td class="text-right pr-4">
-            <a :href="post.guid"
+            <a
+                :href="post.guid"
+                :title="this.post.title"
                 class="no-underline hover:underline font-normal text-grey-darkest"
             >
                 {{ this.post.title }}
             </a>
         </td>
 
         <td class="text-xs text-grey-darker uppercase pr-4">
-            {{ this.post.published_at }}
+            {{ this.published }}
```

Comment:
I don't think I like this attr name; it's unclear what it represents

Reply:
I agree here, I think something like `published_date` would be clearer.